contracts/raffle/Makefile.toml

```diff
@@ -0,0 +1,211 @@
+[env]
+friendbot = "https://friendbot-futurenet.stellar.org/"
+network = "futurenet"
+server = "https://horizon-futurenet.stellar.org/"
+
+[env.futurenet]
+friendbot = "https://friendbot-futurenet.stellar.org/"
+network = "futurenet"
+server = "https://horizon-futurenet.stellar.org/"
+
+[env.standalone]
+friendbot = "http://localhost:8000/friendbot"
+network = "standalone"
+server = "http://localhost:8000"
+
+[tasks.run_standalone_server]
+script = '''
+  docker run --rm -it \
+    -p 8000:8000 \
+    --name stellar \
+    stellar/quickstart:soroban-dev@sha256:a6b03cf6b0433c99f2f799b719f0faadbb79684b1b763e7674ba749fb0f648ee \
+    --standalone \
+    --enable-soroban-rpc
+'''
+
+[tasks.create_account_admin]
+script_runner = "@shell"
+script = '''
+  soroban config identity generate admin 2>/tmp/raffle_error
+  ERR=`cat /tmp/raffle_error`
+  if [ ! -z "$ERR" ]; then
+    echo "error=$ERR"
+    exit 1
+  fi
+  
+  curl "${friendbot}?addr=$(soroban config identity address admin)"
+'''
+
+[tasks.create_account_players]
+script = '''
+  for PLAYER in 1 2 3
+  do
+    soroban config identity generate player_$PLAYER 2>/tmp/raffle_error
+    ERR=`cat /tmp/raffle_error`
```

Comment:
There is a nicer way to store `stderr` in a variable in bash, but for some reason that was the only way that worked when run in a github job.